Cython/Utility/ImportExport.c

```diff
@@ -720,13 +720,25 @@ static int __Pyx_SetVtable(PyObject *dict, void *vtable) {
 
 /////////////// GetVTable.proto ///////////////
 
+#if CYTHON_COMPILING_IN_LIMITED_API
+static void* __Pyx_GetVtable(PyTypeObject *type); /*proto*/
+#else
 static void* __Pyx_GetVtable(PyObject *dict); /*proto*/
+#endif
```

Comment:
It seems that all usage of `__Pyx_GetVtable()` pass `something->tp_dict` as argument. What about always passing this `something`, and, when not using the L'API, extract the `->tp_dict` inside of the function? I think that would avoid a second signature.

Reply:
Done